src/lib/core-extension-service.ts

```diff
@@ -65,16 +70,25 @@ export class CoreExtensionService {
 	 * @param telemetryService
 	 * @param context
 	 */
-	private static async initializeTelemetryService(telemetryService: CoreTelemetryService | undefined, context: vscode.ExtensionContext): Promise<void> {
+	private static async initializeTelemetryService(telemetryService: CoreTelemetryService | undefined, context: vscode.ExtensionContext, outputChannel: vscode.LogOutputChannel): Promise<void> {
 		if (!telemetryService) {
-			console.log(`Telemetry service not present in core dependency API. Using null instead.`);
+			outputChannel.warn(`Telemetry service not present in core dependency API. Using null instead.`);
+			outputChannel.show();
```

Comment:
do we need to warn about the telemetry service instead of logging. Does that outputChannel show the warning to the user on startup? If so, do we care if they know whether or not telemetry is available?
Telemetry is more of a hidden thin in my mind, whereas the apexguru dependencies aren’t. Thoughts?